### DISCUSSION TITLE
is it possible to connect to a remote docker instance?

### DISCUSSION THREAD

**Discussion by nik2208** [microsoft/vscode-docker#3964] [Q&A]
I cannot find any setting to do that

**Comment by karolz-ms** [comment]
Hi @nik2208 , the Docker extension uses the CLI for talking to the Docker daemon, so as long as the CLI (Docker context) is using a remote instance, the extension should use that remote instance automatically. You can even view and switch Docker context directly from the Docker view in VS Code:

![image](https://github.com/microsoft/vscode-docker/assets/15271049/6e0a38d5-a005-49d1-8f81-3075216dfb6c)

Does this answer your question?
